src/libraries/System.Text.Json/gen/System.Text.Json.SourceGeneration.csproj

```diff
@@ -7,6 +7,8 @@
     <NoWarn>CS1574</NoWarn>
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
     <UsingToolXliff>true</UsingToolXliff>
+    <IsNETCoreAppAnalyzer>true</IsNETCoreAppAnalyzer>
+    <AnalyzerLanguage>cs</AnalyzerLanguage>
```

Comment:
Should we default `AnalyzerLanguage` to cs when `IsNETCoreAppAnalyzer` or `IsAspNetCoreAPpAnalyzer` are set to true and `AnalyzerLanguage` is empty?

Reply:
I'd prefer not: https://github.com/dotnet/runtime/pull/52554#discussion_r632060892.
I'd also like folks to force thinking about the fact that they chose to only support `cs`